website/content/docs/nia/release-notes/0-5-0.mdx

```diff
@@ -0,0 +1,356 @@
+  services = ["api", "web"]
+  source   = "path/to/module"
+}
+```
+
+</CodeBlockConfig>
+
+<CodeBlockConfig highlight="4">
+
+```hcl
+task {
+  name     = "services_task"
+  services = ["api", "web"]
+  module   = "path/to/module"
+}
+```
+
+</CodeBlockConfig>
+
+</CodeTabs>
+
+### Deprecate `source_input` block
+
+The `source_input` block in the task configuration is deprecated and can be directly renamed to `module_input`.
+
+Example:
+<CodeTabs heading="Deprecate source_input block" tabs={["Deprecated", "New"]}>
+
+<CodeBlockConfig highlight="7">
+
+```hcl
+task {
+  name   = "scheduled_task"
+  module = "path/to/module"
+  condition "schedule" {
+    cron = "* * * * Mon"
+  }
+  source_input "consul-kv" {
+    path = "my_key"
+  }
+}
+```
+
+</CodeBlockConfig>
+
+<CodeBlockConfig highlight="7">
+
+```hcl
+task {
+  name   = "scheduled_task"
+  module = "path/to/module"
+  condition "schedule" {
+    cron = "* * * * Mon"
+  }
+  module_input "consul-kv" {
+    path = "my_key"
+  }
+}
+```
+
+</CodeBlockConfig>
+
+</CodeTabs>
+
+
+### Deprecate `source_includes_var` field
+
+The `condition` block's `source_includes_var` field is deprecated for all types of conditions and can be directly renamed to `use_as_module_input`.
+
+Example:
+<CodeTabs heading="Deprecate source_includes_var field" tabs={["Deprecated", "New"]}>
+
+<CodeBlockConfig highlight="6">
+
+```hcl
+task {
+  name   = "catalog_services_task"
+  module = "path/to/module"
+  condition "catalog-services" {
+    regexp              = "api"
+    source_includes_var = true
+  }
+}
+```
+
+</CodeBlockConfig>
+
+<CodeBlockConfig highlight="6">
+
+```hcl
+task {
+  name   = "catalog_services_task"
+  module = "path/to/module"
+  condition "catalog-services" {
+    regexp              = "api"
+    use_as_module_input = true
+  }
+}
+```
+
+</CodeBlockConfig>
+
+</CodeTabs>
+
+### Deprecate `services` field
+
+The `services` field can play different roles in a task depending on other task configurations:
+- When no `condition` block is configured, then the `services` field is the task's condition and module input
+- When there is a `condition` block configured, then the `services field is only module input for the task
+
+The `services` field can be replaced depending on the role it has for the task.
+
+**Deprecate `services` field as a condition**
+
+When the `services` field behaves as a condition, it can be replaced with a `condition "services"` block with the `names` field set to the list of the services.
+
+Example:
+<CodeTabs heading="Deprecate services field as a condition" tabs={["Deprecated", "New"]}>
+
+<CodeBlockConfig highlight="4">
+
+```hcl
+task {
+  name     = "services_condition_task"
+  module   = "path/to/module"
+  services = ["api", "web"]
+}
+```
+
+</CodeBlockConfig>
+
+<CodeBlockConfig highlight="4,5,6">
+
+```hcl
+task {
+  name     = "services_condition_task"
+  module   = "path/to/module"
+  condition "services" {
+    names = ["api", "web"]
+  }
```

Comment:
Can we call out in this section that this task is valid because `use_as_module_input` is set to true by default? It may also help to provide an example where we're explicitly setting it to `true`.

Reply:
I realized for `condition "services"` that `use_as_module_input` field was introduced in 0.5. I couldn't get the right wording for a callout here.

I ended up adding `use_as_module_input` in the example below for `condition "consul-kv"` block and added a little callout there. Let me know your thoughts!

Note: apologies for all push noise - ran into some issues with the preview deploy and caught some typos!

Reply:
The callout looks good to me!